playground/frontend/playground_components/lib/widgets/split_view/widget.dart

```diff
@@ -0,0 +1,199 @@
+                      constraints: constraints,
+                      firstPan: _Pan(
+                        child: firstChild,
+                        size: controller.firstSize,
+                      ),
+                      secondPan: _Pan(
+                        child: secondChild,
+                        size: controller.secondSize,
+                      ),
+                      onPanUpdate: controller.onPanUpdate,
+                    ),
+            ),
+          );
+        },
+      ),
+    );
+  }
+}
+
+class _HorizontalLayout extends StatelessWidget {
+  final _Pan firstPan;
+  final _Pan secondPan;
+  final BoxConstraints constraints;
+  final void Function(DragUpdateDetails)? onPanUpdate;
+
+  const _HorizontalLayout({
+    required this.firstPan,
+    required this.secondPan,
+    required this.constraints,
+    required this.onPanUpdate,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: constraints.maxWidth,
+      child: Row(
+        children: <Widget>[
+          SizedBox(
+            width: firstPan.size,
+            child: firstPan.child,
+          ),
+          _Separator(
+            isHorizontal: true,
+            onPanUpdate: onPanUpdate,
+          ),
+          SizedBox(
+            width: secondPan.size,
+            child: secondPan.child,
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _VerticalLayout extends StatelessWidget {
+  final _Pan firstPan;
+  final _Pan secondPan;
+  final BoxConstraints constraints;
+  final void Function(DragUpdateDetails)? onPanUpdate;
+
+  const _VerticalLayout({
+    required this.firstPan,
+    required this.secondPan,
+    required this.constraints,
+    required this.onPanUpdate,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: constraints.maxHeight,
+      child: Column(
+        children: <Widget>[
+          SizedBox(
+            height: firstPan.size,
+            child: firstPan.child,
+          ),
+          _Separator(
+            isHorizontal: false,
+            onPanUpdate: onPanUpdate,
+          ),
+          SizedBox(
+            height: secondPan.size,
+            child: secondPan.child,
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Separator extends StatelessWidget {
+  final bool isHorizontal;
+  final void Function(DragUpdateDetails)? onPanUpdate;
+
+  const _Separator({
+    required this.isHorizontal,
+    required this.onPanUpdate,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return MouseRegion(
+      cursor: isHorizontal
+          ? SystemMouseCursors.resizeLeftRight
+          : SystemMouseCursors.resizeUpDown,
+      child: GestureDetector(
+        behavior: HitTestBehavior.translucent,
+        onPanUpdate: onPanUpdate,
+        child: Container(
+          padding: const EdgeInsets.all(2),
+          width: isHorizontal ? BeamSizes.splitterSize : double.infinity,
```

Comment:
We should either call this `separator` or `splitter` in all cases. `splitViewSeparator` sounds like the clearest one to me.

Reply:
Changed to `splitViewSeparator`.